### PR TITLE
[OSDEV-1388] Own waiter instead waiter from boto3

### PR DESCRIPTION
**Pull request by roninzp**
The waiter from boto3 cannot wait more than half an hour so we replaced it with our own.

### CODE REVIEW

File: deployment/terraform/database_anonymizer_sheduled_task/docker/database_anonymizer.py [hunk 1]
```diff
@@ -112,14 +113,19 @@
...
+    snapshot_status = response['DBSnapshots'][0]['Status']
+
+    if snapshot_status == 'available':
+        print(f"Snapshot {shared_snapshot_identifier} is now available.")
+        break
+    else:
+        print(f"Snapshot {shared_snapshot_identifier} status: {snapshot_status}. Waiting...")
+
+    time.sleep(60)
+    attempt += 1
```
[the remark this answers is not in the document]
Comment (killev) [reply deployment/terraform/database_anonymizer_sheduled_task/docker/database_anonymizer.py:128]: I'd consider suggestion from CodeRabbit useful. Could you please apply it?

[the remark this answers is not in the document]
Comment (roninzp) [reply deployment/terraform/database_anonymizer_sheduled_task/docker/database_anonymizer.py:128]: I don't see any sense in declaring a variable that can only be changed during the deployment process. The constant is currently set to a time greater than the time it takes to generate a snapshot.

**Review by killev — DISMISSED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md [hunk 2]
```diff
@@ -65,6 +65,7 @@ to modify moderation event `status`.
 * The RDS instance for `production` has been upgraded to `db.m6in.4xlarge` and configured to operate in a single Availability Zone.
 
 ### Bugfix
+* [OSDEV-1388](https://opensupplyhub.atlassian.net/browse/OSDEV-1388) - The waiter from boto3 cannot wait more than half an hour so we replaced it with our own.
```
Comment (vladsha-dev) [inline doc/release/RELEASE-NOTES.md:68]: Please create a new section for release version 1.26 and add this information to it. You can use the template from the RELEASE-NOTES-TEMPLATE.md file.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: doc/release/RELEASE-NOTES.md (hunk 2 above)
Comment (roninzp) [reply doc/release/RELEASE-NOTES.md]: done

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Review by VadimKovalenkoSNF — APPROVED** [review]
Approved.

**Pull request merged**